### PR TITLE
SRVCOM-1728: Updating autoscaling docs abstracts for Jupiter

### PR DESCRIPTION
**Pull request by abrennan89**
Jira: https://issues.redhat.com/browse/SRVCOM-1728

Applies for OCP 4.6+

Direct preview link:
https://deploy-preview-44138--osdocs.netlify.app/openshift-enterprise/latest/serverless/develop/serverless-autoscaling-developer.html

### CODE REVIEW

**Comment by nak3** [comment]
LGTM

We need to update the annotation by `-` hyphen style like `min-scale` and `max-scale` :sweat_drops: (The old style still works so we can do it by another PR.)

**Comment by jrangelramos** [comment]
QE Approved

**Review by nalhadef — COMMENTED** [review]
Looks good; a couple of minor nits.

File: modules/serverless-autoscaling-developer-minscale.adoc [hunk 1]
```diff
@@ -6,16 +6,14 @@
 [id="serverless-autoscaling-developer-minscale_{context}"]
 = Minimum scale bounds
 
-The minimum number of replicas that can serve an application is determined by the `minScale` annotation.
+The minimum number of replicas that can serve an application is determined by the `minScale` annotation. If scale to zero is not enabled, the `minScale` value defaults to `1`.
```
[the remark this answers is not in the document]
Comment (abrennan89) [reply modules/serverless-autoscaling-developer-minscale.adoc:9]: Nah it's a functionality :) see https://www.suse.com/c/do-you-really-need-to-scale-to-zero/

**Pull request merged**